### PR TITLE
Update PhantomJS to 2.1.1, remove 2.0 warning in readme

### LINKED ISSUE #437: Update to PhantomJS 2.1
PhantomJS 2.1 was (finally) released today: https://groups.google.com/d/msg/phantomjs/Jp4sDXrfmNg/I5J5TBFBDQAJ

I'm interested in updating [grunt-contrib-qunit](https://github.com/gruntjs/grunt-contrib-qunit) to PhantomJS 2.1, this is the end of the dependency chain.

I'll look into creating a pull request for the update.

### PR DESCRIPTION
**Pull request by jzaefferer**
Fixes #437

### CODE REVIEW

**Comment by mikesherov** [comment]
Need to update the link location and checksums as well in the install.js script. I was going to do this when I got into work in an hour ;-)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "phantomjs",
-  "version": "1.9.19",
+  "version": "2.1.1",
```
Comment (mikesherov) [inline package.json]: Why wouldn't this be 2.0.0?

Comment (jzaefferer) [inline package.json]: From the readme: "The major and minor number tracks the version of PhantomJS that will be installed. The patch number is incremented when there is either an installer update or a patch build of the phantom binary." I guess 2.1.0 would also be okay.

**Comment by jzaefferer** [comment]
> Need to update the link location and checksums as well in the install.js script.

These? https://github.com/Medium/phantomjs/blob/master/install.js#L384-L394 Do you know where to get the checksums from?

Feel free to replace this PR with your own, from scratch or with this as base.

File: package.json (hunk 1 above)
Comment (Arkni) [inline package.json]: > I guess 2.1.0 would also be okay.

There is no binaries available for v2.1.0. So v2.1.1 is the only choice.

**Comment by Arkni** [comment]
> Do you know where to get the checksums from?

From here: http://phantomjs.org/download.html
There is an MD5 Checksums' section (which is the one used in this lib).

File: package.json (hunk 1 above)
Comment (mgol) [inline package.json]: > There is no binaries available for v2.1.0. So v2.1.1 is the only choice.

The readme snippet @jzaefferer mentioned says only major & minor needs to agree with PhantomJS so 2.1.0 would be fine.

Comment (Arkni) [inline package.json]: My apologies, I misunderstood the used process. I get it now :)
Thanks @mgol :)

**Comment by mikesherov** [comment]
https://github.com/Medium/phantomjs/pull/439
 replaces this

**Pull request closed without merging**